reference/service/collaboration.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Collaboration
+---
+
+The Pulumi Cloud Console offers multiple that people can collaborate on a
+stack. However, some may not be available depending on the type of organization
+the stack resides in.
+
+## Teams
+
+Teams allow organization admins to assign permissions to access a group of stacks
+to a group of users.
+
+The ability to create teams is limited to Pulumi organizations imported from
```

Comment:
I would phrase this differently. "The ability to create teams in the Community and Standard Editions of the service is limited to organizations imported from GitHub. The Enterprise Edition supports alternative identity providers."

Reply:
Sure, done.